src/windows/find_tools.rs

```diff
@@ -509,6 +518,45 @@ mod impl_ {
         find_tool_in_vs16plus_path(r"MSBuild\Current\Bin\MSBuild.exe", target, "16", env_getter)
     }
 
+    pub(super) fn find_llvm_tool(
+        tool: &str,
+        target: TargetArch,
+        env_getter: &dyn EnvGetter,
+    ) -> Option<Tool> {
+        find_llvm_tool_vs17(tool, target, env_getter)
```

Comment:
While Clang/LLVM [can be installed as of Visual Studio 2019 version 16.2](https://learn.microsoft.com/en-us/cpp/build/clang-support-msbuild?view=msvc-170), I just went with Visual Studio 2022 support for now.